src/reviewer.py

```diff
@@ -23,11 +23,8 @@ def review_pull_request(self):
                 return
 
 
-            # Post all comments in a single batch
-            if review_comments:
-                post_review_comment(review_comments, diff_content)
-            else:
-                print("No valid review comments generated")
+            print(f"Generated {len(review_comments)} potential comments")
+            post_review_comment(review_comments, diff_content)
 
         except Exception as e:
             print(f"Review failed: {str(e)}")
```

Comment:
Changed the logic to print the number of potential comments and call the post_review_comment function.
```suggestion
|
```
N/A